compiler/frontend/spark/src/main/java/org/apache/nemo/compiler/frontend/spark/transform/FlatMapTransform.java

```diff
@@ -61,6 +62,11 @@ public void onWatermark(final Watermark watermark) {
     outputCollector.emitWatermark(watermark);
   }
 
+  @Override
+  public void onLatencymark(final Latencymark latencymark) {
```

Comment:
Lots of duplicate `onLatencyMark` code blocks. It would be good to refactor it.

Reply:
Perhaps we can create an abstract class that contains 61-68 code blocks and inherit the abstract class here

Reply:
done